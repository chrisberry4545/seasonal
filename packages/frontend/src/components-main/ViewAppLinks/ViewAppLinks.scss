@import '../../styles/settings';
@import '../../styles/tools';

.c-view-app-links {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: space-between;
  min-height: 190px;

  @include mq($from: tablet) {
    flex-direction: row-reverse;
    min-height: 78px;
    max-width: 570px;
    margin: auto;
  }
}
