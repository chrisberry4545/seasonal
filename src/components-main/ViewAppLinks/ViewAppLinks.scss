@import './../../styles/settings';
@import './../../styles/tools';

.c-view-app-links {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: space-between;
  min-height: 150px;

  @include mq($from: tablet) {
    flex-direction: row-reverse;
    min-height: 0;
    max-width: 400px;
    margin: auto;
  }
}
