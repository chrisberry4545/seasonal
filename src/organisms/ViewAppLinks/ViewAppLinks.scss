@import './../../styles/settings';
@import './../../styles/tools';

.c-view-app-links {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: space-between;
  min-height: 150px;

  @include mq($from: tablet) {
    min-height: 0;
    flex-direction: row;
    max-width: 400px;
    margin: auto;
  }
}
