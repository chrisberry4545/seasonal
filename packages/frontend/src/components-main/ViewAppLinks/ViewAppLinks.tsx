import React, { FC } from 'react';
import './ViewAppLinks.scss';
import {
  AndroidAppDownload
} from '../AndroidAppDownload/AndroidAppDownload';
import {
  ViewWebAppVersionConnector
} from '../ViewWebAppVersion/ViewWebAppVersion.connector';
import { IOSAppDownload } from '../IOSAppDownload/IOSAppDownload';

export const ViewAppLinks: FC<{}> = () => (
  <div className='c-view-app-links'>
    <IOSAppDownload />
    <AndroidAppDownload />
    <ViewWebAppVersionConnector />
  </div>
);
