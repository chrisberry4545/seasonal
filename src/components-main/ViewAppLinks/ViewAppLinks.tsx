import React, { SFC } from 'react';

import './ViewAppLinks.scss';

import {
  AndroidAppDownload
} from '../AndroidAppDownload/AndroidAppDownload';

import {
  ViewWebAppVersionConnector
} from '../ViewWebAppVersion/ViewWebAppVersion.connector';

export const ViewAppLinks: SFC<{}> = () => (
  <div className='c-view-app-links'>
    <ViewWebAppVersionConnector />
    <AndroidAppDownload />
  </div>
);
