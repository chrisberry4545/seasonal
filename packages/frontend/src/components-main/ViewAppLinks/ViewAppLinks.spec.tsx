
import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import { ViewAppLinks } from './ViewAppLinks';

jest.mock('../AndroidAppDownload/AndroidAppDownload', () => ({
  AndroidAppDownload: () => 'AndroidAppDownload'
}));
jest.mock('../ViewWebAppVersion/ViewWebAppVersion.connector', () => ({
  ViewWebAppVersionConnector: () => 'ViewWebAppVersionConnector'
}));
jest.mock('../IOSAppDownload/IOSAppDownload', () => ({
  IOSAppDownload: () => 'IOSAppDownload'
}));

describe('<ViewAppLinks />', () => {
  let wrapper: ShallowWrapper;

  beforeEach(() =>
    wrapper = shallow(
       <ViewAppLinks />
    )
  );

  test('renders correctly', () => expect(wrapper).toMatchSnapshot());

});
