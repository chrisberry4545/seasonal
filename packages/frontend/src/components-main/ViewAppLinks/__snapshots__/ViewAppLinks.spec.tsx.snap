// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ViewAppLinks /> renders correctly 1`] = `
<div
  className="c-view-app-links"
>
  <IOSAppDownload />
  <AndroidAppDownload />
  <ViewWebAppVersionConnector />
</div>
`;
